examples/benchmark/scene_test_2.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+# vispy: testskip
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Compare an optimal plot grid implementation to the same functionality
+provided by scenegraph.
```

Comment:
Can you add comments to these files with helpers suggesting how to profile?

Also, I get some errors on this example, does it work for you?

```
  File "examples/benchmark/scene_test_2.py", line 49, in on_draw
    M = len(self.cells)
AttributeError: 'GridCanvas' object has no attribute 'cells'
```